pkg/dnsman2/dns/provider/handler/aws/handler.go

```diff
@@ -152,7 +153,8 @@ func (h *handler) GetZones(ctx context.Context) ([]provider.DNSHostedZone, error
 
 		output, err := paginator.NextPage(ctx)
 		if err != nil {
-			return nil, err
+			log.Info("error listing hosted zones", "error", err)
```

Comment:
Semantically we should log this as an error
```suggestion
			log.Error(err, "error listing hosted zones")
```